tools/settings.py

```diff
@@ -261,7 +261,7 @@ def __setattr__(self, name, value):
     self.attrs[name] = value
 
   def check_type(self, name, value):
-    if name in ('SUPPORT_LONGJMP', 'PTHREAD_POOL_SIZE', 'SEPARATE_DWARF', 'LTO', 'SINGLE_FILE', 'SUPPORT_BASE64_EMBEDDING'):
```

Comment:
Maybe put this (and the settings.js change) into a separate PR?  (To keep this one focused)

Reply:
Oh no, this change just revert to the very original file because I added single_file and support_base64_embedding for the =2 thing, but now that it is removed, I just revert it back to the original file.